### PR TITLE
[CCTRI-2096] CME API

### PR DESCRIPTION
**Pull request by mstoro**

### CODE REVIEW

File: api/schemas.py [hunk 1]
```diff
@@ -12,3 +12,17 @@ class PayloadSchema(Schema):
...
+
+
+class CMEPayloadSchema(Schema):
+    startDate = fields.Date(format='%Y-%m-%d',
+                            required=True)
+    endDate = fields.Date(format='%Y-%m-%d',
+                          required=True)
+
+    @validates_schema()
+    def validate_date(self, data, **kwargs):
```
Comment (AnastasiiaRozlyvan) [inline api/schemas.py]: I find it a bit confusing that this file has `validate_date` names repetitive.

Comment (mstoro) [reply api/schemas.py]: https://github.com/mstoro/nasa_api/pull/3/commits/ab6cdcf191a46490a7c5e6356d9905d10d2be676

File: api/schemas.py [hunk 2]
```diff
@@ -12,3 +12,17 @@ class PayloadSchema(Schema):
...
+
+class CMEPayloadSchema(Schema):
+    startDate = fields.Date(format='%Y-%m-%d',
+                            required=True)
+    endDate = fields.Date(format='%Y-%m-%d',
+                          required=True)
+
+    @validates_schema()
+    def validate_cme_period(self, data, **kwargs):
+        if not (data['startDate'] <= data['endDate']):
```
Comment (AnastasiiaRozlyvan) [inline api/schemas.py]: Can we rewrite this condition without using negation to improve code readability?

File: api/utils.py [hunk 3]
```diff
@@ -43,3 +43,26 @@ def get_json(scheme):
...
+
+def cme_form_data(data):
+    if isinstance(data, list):
+        res = {}
+        for element in data:
+            res['note'] = element['note']
+            res['startDate'] = element['startTime']
+            res['info'] = element['cmeAnalyses']
+
+        return res
```
Comment (MaxMhonc) [inline api/utils.py]: Looks like if data will be list of dicts, your function will return only last dict. 
Maybe should use:
res = []
for element in data:
            res.append({ ... })

File: api/utils.py [hunk 4]
```diff
@@ -43,3 +43,26 @@ def get_json(scheme):
         raise RequestError(errors)
 
     return data
+
+
+def cme_form_data(data):
```
Comment (MaxMhonc) [inline api/utils.py]: Good practice is to use firstly verb in function name in this case.

File: api/schemas.py (hunk 2 above)
Comment (mstoro) [reply api/schemas.py]: https://github.com/mstoro/nasa_api/pull/3/commits/1ee43f97545851be885052bf454acdd6ed5dc6c9

File: api/utils.py (hunk 3 above)
Comment (mstoro) [reply api/utils.py]: https://github.com/mstoro/nasa_api/pull/3/commits/5ab6719a8dbb63b3ffec202d5e189127bb3357f6

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by MaxMhonc — APPROVED** [review]

**Pull request merged**